src/gt4py/next/common.py

```diff
@@ -844,18 +844,20 @@ def is_connectivity_field(
 
 
 @functools.singledispatch
-def field(
+def _field(
     definition: Any,
     /,
     *,
     domain: Optional[DomainLike] = None,
     dtype: Optional[core_defs.DType] = None,
 ) -> Field:
+    # Utility function to construct a `Field` from different buffer representations.
+    # Consider removing this function and using `Field` constructor directly.
```

Comment:
Put the comment above the function (and optionally mention also `_connectivity`)